### PR TITLE
Assignment 5 skeleton check-in

### PR DESCRIPTION
**Pull request by cmalik3595**
Please review. Thanks !

### CODE REVIEW

File: assignment5/cat_correlation.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        numpy.multiply(
+            n_array, numpy.power(numpy.subtract(y_avg_array, y_total_avg), 2)
+        )
+    )
+    denominator = numpy.sum(numpy.power(numpy.subtract(values, y_total_avg), 2))
+    if numerator == 0:
+        eta = 0.0
+    else:
+        eta = numpy.sqrt(numerator / denominator)
+    return eta
```
Comment (Anujagagangras) [inline assignment5/cat_correlation.py:120]: please add correlation plots.

File: assignment5/pca.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+    pc_n = len(var_df.index)
+
+    pca_var = pd.DataFrame(pca.components_, columns=preds_norm.columns)
+
+    pca_var = pca_var[:pc_n].T.abs()
+
+    pca_var = pca_var.style.background_gradient(cmap="viridis_r", axis=0)
+
+    return pca_var, var_df
```
Comment (Anujagagangras) [inline assignment5/pca.py:27]: add one more different model. Divide the data by train test split.

Comment (cmalik3595) [reply assignment5/pca.py:27]: Data is already divided in 70:30 ratio. Will add PCA model

File: assignment5/cat_correlation.py (hunk 1 above)
Comment (cmalik3595) [reply assignment5/cat_correlation.py:120]: Added

[the remark this answers is not in the document]
Comment (cmalik3595) [reply assignment5/cat_correlation.py:120]: added

**Pull request merged**